src/layouts/layout.jsx

```diff
@@ -118,6 +120,18 @@ function Layout({ children }) {
     return null;
   };
 
+  useEffect(() => {
+    if (filtersLinksRef.current && loaded) {
+      const offsetTop = filtersLinksRef.current.offsetTop;
+      window.scrollTo({ behavior: 'smooth', top: offsetTop - 120 }); 
```

Comment:
Just a doubt, why -120?

Reply:
Because the menu cover the filters and you need to make a space between the menu and the top of the page